doc/tutorials/run_graph_states.py

```diff
@@ -205,3 +207,31 @@
 
 viz.plot_params["figure.figsize"] = (5.4, 4)
 viz.plot_mat_heat_map(adj)
+
```

Comment:
Let us please do the following:

- Move the adjacency matrix part to after the complete graph so we have a better visual.
- Replace `viz.plot_mat_heat_map(adj)` with GHZ_state.draw_adj().
- Add a line that says: "As expected, we have black tiles on the diagonal of the heat map, indicating no self connections (loops), and yellow tiles everywhere else, indicating maximal connectedness."
- Replace line 215 or `egraph.py` with `adj = self.adj_generator(sparse=False)` (since it it is equivalent but shorter).
- Change the thumbnail of the tutorial (in the _static directory) to the complete graph plot.